common/src/main/java/org/dash/wallet/common/ui/BaseLockScreenFragment.kt

```diff
@@ -1,28 +0,0 @@
-package org.dash.wallet.common.ui
-
-import android.os.Bundle
-import android.util.Log
-import android.view.View
-import androidx.appcompat.app.AlertDialog
-import androidx.fragment.app.DialogFragment
-import androidx.fragment.app.Fragment
-import dagger.hilt.android.AndroidEntryPoint
-import org.dash.wallet.common.services.LockScreenBroadcaster
-import javax.inject.Inject
-
-@AndroidEntryPoint
-@Deprecated("Use AdaptiveDialog or derive your dialog from DialogFragment have it auto-dismissible")
-open class BaseLockScreenFragment: Fragment() {
```

Comment:
No longer needed, dialogs are replaced with `AdaptiveDialog` in former descendants.